tailwind.config.js

```diff
@@ -25,7 +26,11 @@ module.exports = {
         red: "#FF4B1E",
       },
     },
-    extend: {},
+    extend: {
+      transitionProperty: {
+        width: "width",
```

Comment:
what is this for?

Reply:
it was supposed to be for the circle to pill shape transition, but i just found out it did nothing 😭. i'll delete it.